README.md

```diff
@@ -28,3 +28,34 @@ Using a command line run
 npm install
 npm start
 ```
+
+## How to add or update screenshots
+
+Screenshots for service landing or home pages can be added automatically by running a script.
+
+First, you need to have the latest version of [Google Chrome](https://www.google.com/chrome/) installed.
+
+To tell the script where the Google Chrome executable is installed on your computer, add the full path to a file named `.env` (in the same folder as this README file), using this
+named environment variable:
```

Comment:
```suggestion
To tell the script where the Google Chrome executable is located on your computer, create a file named `.env` (in the same folder as this `README.md` file) and add this named environment variable:
```

Clearer distinction between adding the `.env` file and adding the variable. Also include content on one line so easier to diff.